src/wincameranode.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#include "wincapture.h"
+#include "winrospublisher.h"
+#include <ros/ros.h>
+#include <string>
+using namespace winrt::Windows::System::Threading;
+using namespace winrt::Windows::Foundation;
+using namespace ros_win_camera;
+const int32_t PUBLISHER_BUFFER_SIZE = 4;

```

Comment:
Just curious why the buffer size is hard-coded as 4 here? Is it something should be configured by the users?